File: src/Model/FileStatement/FileData.java
package Model.FileStatement;

import java.io.BufferedReader;

public class FileData {
    private String fileName;
    private BufferedReader reader;

    public FileData(String fileName, BufferedReader reader) {
        this.fileName = fileName;
        this.reader = reader;
    }

    public String getFileName() {
        return fileName;
    }

    public void setFileName(String fileName) {
        this.fileName = fileName;
    }

    public BufferedReader getReader() {
        return reader;
    }

    public void setReader(BufferedReader reader) {
        this.reader = reader;
    }

    @Override
    public String toString() {
        return fileName;
    }
}

File: Toy-Language-C#-Intepreter/Toy-Language-CS-Interpreter/Toy-Language-CS-Interpreter/Model/Statement/FileStatement/FileData.cs
using System;
using System.Collections.Generic;
using System.IO;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Toy_Language_CS_Interpreter.Model.Statement.FileStatement
{
    class FileData
    {
        private string fileName;
        private StreamReader reader;

        public FileData(string fileName, StreamReader reader)
        {
            this.fileName = fileName;
            this.reader = reader;
        }
        public string FileName
        {
            get { return fileName; }
            set { fileName = value; }
        }

        public StreamReader Reader
        {
            get { return reader; }
            set { reader = value; }
        }
        public override string ToString()
        {
            return fileName;
        }
    }
}
